### PR TITLE
Add devcontianer settings

### PR DESCRIPTION
**Pull request by meganetaaan**
Added devcontainer settings for VSCode

* C++ preset
* clone and build [Dynamixel SDK](https://github.com/ROBOTIS-GIT/DynamixelSDK) at `/external` directory

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/cpp/.devcontainer/base.Dockerfile
+
+# [Choice] Debian / Ubuntu version: debian-10, debian-9, ubuntu-20.04, ubuntu-18.04
+ARG VARIANT="buster"
```
Comment (ShotaAk) [inline .devcontainer/Dockerfile]: Could you please change this to `ubuntu 20.04`?
The uninstall script didn't work properly on debian-10. It's probably due to a different version of cmake.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/cpp
+{
+	"name": "C++",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Debian / Ubuntu OS version: debian-10, debian-9, ubuntu-20.04, ubuntu-18.04
+		"args": { "VARIANT": "debian-10" }
```
Comment (ShotaAk) [inline .devcontainer/devcontainer.json]: Could you please change this to `ubuntu 20.04`?

File: build_install.bash [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/bash -e
```
Comment (ShotaAk) [inline build_install.bash]: Please remove this change after switching the environment to `ubuntu 20.04`.

File: max_e1_examples/CMakeLists.txt [hunk 4]
```diff
@@ -7,6 +7,9 @@ project(max_e1_examples VERSION 1.0)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
+include_directories(../max_e1_lib/include)
+link_directories(../max_e1_lib/build)
+
```
Comment (ShotaAk) [inline max_e1_examples/CMakeLists.txt:12]: Thanks!

**Review by ShotaAk — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please check my comments.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (meganetaaan) [reply .devcontainer/Dockerfile]: Fixed to Ubuntu 20.04 and confirmed that it works.

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (meganetaaan) [reply .devcontainer/devcontainer.json]: Fixed.

File: build_install.bash (hunk 3 above)
Comment (meganetaaan) [reply build_install.bash]: Fixed.

**Comment by meganetaaan** [comment]
All requested changes have been fixed. Please confirm them.

**Review by ShotaAk — APPROVED** [review]
LGTM

**Pull request merged**